drivers/ukintctlr/gic/gic-v3.c

```diff
@@ -683,15 +682,11 @@ static int gicv3_do_probe(void)
 		return r;
 	}
 
-	in.dist_addr = gicv3_drv.dist_mem_addr;
-	in.rdist_addr = gicv3_drv.rdist_mem_addr;
-	r = uk_intctlr_plat_probe(&in, &out);
+	r = uk_intctlr_plat_probe(&gicv3_drv);
 	if (unlikely(r)) {
 		uk_pr_err("GICv3 initialization error");
```

Comment:
Maybe it makes sense to rephrase message here as it was done for Gic-v2

Reply:
Thanks, fixed 👍🏼